backend/src/main/java/com/eyetrade/backend/service/RegistrationService.java

```diff
@@ -49,7 +49,8 @@ public CompleteUserResource saveTraderUser(TraderUserDto traderRegisterDto, Stri
     @Transactional
     private CompleteUserResource save(User user) {
         userRepository.saveAndFlush(user);
-        confirmationTokenService.sendActivationToken(user);
```

Comment:
This can be about pc. Because, when I tried in my pc and in cloud, I didn't get error.

Reply:
Okay then, so we need to test it again.